capture.md

```diff
@@ -0,0 +1,111 @@
+# _Capture_
+
+The **_first_ step** in creating
+a **system** of for personal effectiveness
+is to help people ***capture***
+_everything_ on their mind. <br />
+Emptying one's mind (_specifically working memory_)
+of everything that is occupying space and processing power
+is _essential_ for gaining focus
+on the _one_ most important priority.
+This step is often referred to as a "brain dump".
+
+<div align="center">
+    <a href="https://github.com/dwyl/product-roadmap/issues/12">
+        <img src="https://user-images.githubusercontent.com/194400/68108635-3497be00-fee0-11e9-83f0-6a56ecc99623.jpg" width="500">
+    </a>
+</div>
+<!-- this diagram is borrowed from https://anniemueller.com/brain-dump-noun-1
+we definitely need a better one. If you want to help with this,
+please open an issue: github.com/dwyl/product-roadmap/issues/new -->
+
+
+> A brain dump is the act of writing down everything
+that is on your mind so that you don't forget it
+and to clear your brain to **focus** on **one thing**
+instead of being distracted by many things.
+https://anniemueller.com/brain-dump-noun-1
+
+## Free Text Input
+
+The first feature we need to build
+is a way of capturing all thoughts
+on one's mind.
+Thankfully this is just a matter
+of capturing free text (_using a **`textarea`** element_)
+and safely storing it as **`text`** in Postgres.
+
+Figma wireframe: https://www.figma.com/proto/WrpkKGJNYZbeCzMRDVtvQLbC/dwyl-app
```

Comment:
How do we signal that we have _finished_ typing and want to move onto the next item?

Reply:
@iteles _great_ question! Please capture it in an issue: https://github.com/dwyl/app/issues